iperf-post-process

```diff
@@ -139,8 +139,21 @@ sub process_proto {
                     $bitrate = $columns[6];
                     debug_print("bitrate $bitrate\n");
 
-                    if ( $rateunit eq 'none' ) {
-                        $rateunit = $columns[7];
```

Comment:
We only need to determine the rateunit once per section.  Could save some cpu cycles by not reread the same column..